src/PatientBrowserTutorial/style.css

```diff
@@ -0,0 +1,49 @@
+@font-face {
```

Comment:
I am not in favor of the custom styling. I think it distracts from making a simple clean example.